### ISSUE TITLE
Readme link to vmr-sync script is broken

### ISSUE THREAD

**Issue by mthalman** [dotnet/source-build#3976]
In the context of the VMR, the [link to the vmr-sync.sh script is broken on the main readme](https://github.com/dotnet/dotnet?tab=readme-ov-file#synchronizing-code-into-the-vmr). The link will have to be hardcoded and not relative. The question would be which repo it should point to: installer or VMR?

And just to confirm, can this script be used in any context, not just the installer repo?

**Comment by MichaelSimons** [comment]
> And just to confirm, can this script be used in any context, not just the installer repo?

Yes it can run for any repo.  The help provides an [example](https://github.com/dotnet/installer/blob/main/eng/vmr-sync.ps1#L23) of this.
